definitions/hackingai.json

```diff
@@ -0,0 +1,22 @@
+{
+
+    "mlflow": {
+        "cmdline": ["mlflow"],
+        "domain": ["mlflow.org"]
+    },
+    "h2o.ai": {
+        "process_name": ["java*",
+                         "hadoop*"],
```

Comment:
Recommend removing these two processes since they aren't unique to the `h2o.ai` application
```suggestion
```